demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -86,14 +86,42 @@ def get_playbook_dependencies(playbook):
     playbooks = set()
 
     playbook_tasks = playbook.get('tasks')
+    playbook_path = os.path.relpath(input)
+    pack_path = os.path.dirname(os.path.dirname(playbook_path))
+    integration_dir_path = os.path.join(pack_path, 'Integrations')
+    # Get all files in integrations directories
+    pack_files = [os.path.join(r, file) for r, d, f in os.walk(integration_dir_path) for file in f]
+    integrations_files = []
+    for file in pack_files:
+        if file.endswith('.yml') and not os.path.basename(file).startswith('integration'):
+            # Get all yml files
+            integrations_files.append(file)
     for task in playbook_tasks:
         task = playbook_tasks[task]['task']
         if task['iscommand']:
             integration = task['script']
+            brand_integration = task['brand']
             integration = integration.split('|||')
             if integration[0]:
                 integrations.add(integration[0])
-            commands.add(integration[1])
+                if 'Builtin' in integrations:
+                    integrations.remove('Builtin')
+                commands.add(integration[1])
+
+            elif brand_integration:
+                integrations.add(brand_integration)
+
+            elif 'Packs' in playbook_path:
+                for file_ in integrations_files:
+                    with open(file_) as f:
+                        if integration[1] in f.read():
+                            integration_dependency_path = os.path.dirname(file_)
+                            integration_dependency = os.path.basename(integration_dependency_path)
+                            # Case of old integrations without a package.
+                            if integration_dependency == 'Integrations':
```

Comment:
dont we need to check if the command is implemented in that integration?

Reply:
We are checking. I'm going over all the yml files in the list and checking if the command is there or not. 
If it's an old format of the integration, without package, i'll take just the name of the integration.